db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_04_140723) do
+ActiveRecord::Schema.define(version: 2020_05_10_184356) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "answers", force: :cascade do |t|
+    t.text "answer_text"
+    t.bigint "question_id"
+    t.boolean "correct", default: false, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["question_id"], name: "index_answers_on_question_id"
```

Comment:
if we do https://github.com/solocoinapp/solocoin-backend/pull/69/files#r422975083 we shouldn't need this